apps/andi/lib/andi/services/dataset_disable_service.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Andi.Services.DatasetDisableService do
```

Comment:
if we envision this being a recurring pattern for grouping code, would it make sense to just name this module `Andi.Services.DatasetDisable`? or `Andi.Services.DisableDataset` if you prefer.